templates/hooks/cluster/sdk_read_many_post_set_output.go.tpl

```diff
@@ -27,4 +27,11 @@
 	respErr := rm.setAllowedNodeTypeUpdates(ctx, ko)
 	if respErr != nil {
 		return nil, respErr
-	}
\ No newline at end of file
+	}
+
+	resourceARN := (*string)(ko.Status.ACKResourceMetadata.ARN)
+    tags, err := rm.getTags(ctx, *resourceARN)
```

Comment:
Indentation needs fixing here

Reply:
Not sure why this happened. The indentation looks good in my local file.

Reply:
Github bug I think